src/PowerShellEditorServices/Services/Symbols/PesterDocumentSymbolProvider.cs

```diff
@@ -15,10 +15,11 @@ namespace Microsoft.PowerShell.EditorServices.Services.Symbols
     /// Provides an IDocumentSymbolProvider implementation for
     /// enumerating test symbols in Pester test (tests.ps1) files.
     /// </summary>
-    public class PesterDocumentSymbolProvider : IDocumentSymbolProvider
+    internal class PesterDocumentSymbolProvider : IDocumentSymbolProvider
     {
+        string IDocumentSymbolProvider.ProviderId => nameof(PesterDocumentSymbolProvider);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private property 'IDocumentSymbolProvider.ProviderId'.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4979332)